packages/services/api/src/modules/schema/providers/schema-manager.ts

```diff
@@ -685,6 +686,16 @@ export class SchemaManager {
       schemaCheckId,
     );
 
+    if (isUUID(schemaCheckId) === false) {
```

Comment:
`schemaCkeckId` is user input and can be a non-uuid if people mess around with the URL in the browser.